graphql_compiler/tests/test_data_tools/integration_data/create_orientdb_integration.sql

```diff
@@ -0,0 +1,5 @@
+# Copyright 2018-present Kensho Technologies, LLC.
+create vertex Animal set name = 'Animal 1', uuid = 'cfc6e625-8594-0927-468f-f53d864a7a50'
+create vertex Animal set name = 'Animal 2', uuid = 'cfc6e625-8594-0927-468f-f53d864a7a51'
+create vertex Animal set name = 'Animal 3', uuid = 'cfc6e625-8594-0927-468f-f53d864a7a52'
+create vertex Animal set name = 'Animal 4', uuid = 'cfc6e625-8594-0927-468f-f53d864a7a53'
```

Comment:
Is the plan in a future PR to make the integration test data converge with the snapshot test data? Otherwise, we'd be maintaining two separate input data sets. And if they converge, then the snapshot tests start working for all databases too.

Reply:
That would be great, maintaining two is not ideal. The value I see in this current approach is that the results can be reasoned about. The dataset is small and tractable since we generate it ourselves.

On the snapshot side, a failure between backends is going to be somewhat difficult to debug if there were a discrepancy because the resultant dataset can be hard to reason about, given that it was randomly generated. To unify then, perhaps we could revisit the snapshot dataset, and create a non-random version?

Reply:
I think a hand-curated dataset would be hard to maintain as we add features to the compiler, and may become a bottleneck to development. I think of these tests as the last line of defense -- if something fails at this level, it indicates a missing unit test. Since such failures shouldn't happen often (or, ideally, at all), I'm not that worried about having to start from a complex randomly-generated test case and try to work out a minimal repro, and I'd prefer to make it easier to develop new features instead.

What do you think?

Reply:
With the SQL backend, I was planning on using hand-curated data at the unit-test level. An easy way to test was based on using Sqlite, compiling and running a query to get expected results back. From there it seemed natural to generalize and test across other relational DBs + OrientDB while I was at it. Especially since we don't do this sort of testing yet beyond reg tests with the snapshots. 

I do agree that features to the compiler, especially for MATCH should not be gated based on having hand-curated data, but I expect most SQL features will get it regardless unless there's an alternative way to test the SQL backend. Could the scope of tests like this then be limited to the features both backends support?

Reply:
Might be best to sync up on this sometime tomorrow and figure out the best way forward, as I'd really like to unblock this ASAP.

Reply:
I figured out a unit-test method, I'll make a separate PR for that setup.